packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -410,26 +410,19 @@ skipInNode(describe)('plugin-meetings', () => {
             })
         ])));
 
-      it('alice shares the screen with highFrameRate', () => alice.meeting.getMediaStreams({
-        sendShare: true
-      }, {}, {highFrameRate: true})
-        .then((response) => Promise.all([
-          testUtils.delayedPromise(alice.meeting.updateShare({
-            sendShare: true,
-            receiveShare: true,
-            stream: response[1]
-          })),
-          testUtils.waitForEvents([{scope: alice.meeting, event: 'meeting:startedSharingLocal'}]),
-          testUtils.waitForEvents([{scope: bob.meeting.members, event: 'members:update'}])
-            .then((response) => {
-              console.log('SCREEN SHARE RESPONSE ', JSON.stringify(response));
-            }),
-          testUtils.waitForEvents([{scope: alice.meeting, event: 'media:ready'}])
-            .then((response) => {
-              console.log('MEDIA:READY event ', response[0].result);
-              assert.equal(response[0].result.type === 'localShare', true);
-            })
-        ]))
+      it('alice shares the screen with highFrameRate', () => Promise.all([
+        testUtils.delayedPromise(alice.meeting.shareScreen({sharePreferences: {highFrameRate: true}})),
```

Comment:
Did you mean to leave out 'sendShare' and 'sendAudio' from your parameter object?

Reply:
`shareScreen` defaults `sendShare` and `sendAudio` appropriately so it is not needed here.

Reply:
Do you mean 'Media.getDisplayMedia' defaults correctly. Because this object passed into 'shareScreen' will override the one set by default.

Reply:
Oh, I see what you mean, I need to update the defaults code. Good catch!

Reply:
@mwegman I've updated the code